iron-query-params.html

```diff
@@ -52,10 +59,16 @@
       this._dontReact = false;
     },
 
+    detached: function() {
+      this.__ignoreLazy = false;
+    },
+
     paramsObjectChanged: function() {
-      if (this._dontReact) {
+      if (this._dontReact || (this.lazyLoaded && !this.__ignoreLazy)) {
```

Comment:
You mentioned wanting to find a different name earlier, I think you're right: the names make this line very confusing. I haven't thought of any good alternatives so far but I think we should find one.

Reply:
* `lazyLoaded`
* `lazyAttached`
* `ignoreFirstRewrite`
* `ignoreFirstChange`
* `hackyMcHackerson3000`